src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -10805,5 +10805,44 @@ public async Task FormatRawStringInterpolation2()
 var s = $""""""{s, 0 : x }""""""
 ");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task AnonymousMethodIntendBraces_01()
+        {
+            var options = new OptionsCollection(LanguageNames.CSharp)
+            {
+                { IndentBraces, true },
+                { NewLinesForBracesInAnonymousMethods, true }
+            };
+
+            var code = @"
+delegate () {
+}";
+            var expected = @"
+delegate ()
+    {
+    }";
+            await AssertFormatAsync(expected, code, changedOptionSet: options);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task AnonymousMethodIntendBraces_02()
+        {
+            var options = new OptionsCollection(LanguageNames.CSharp)
+            {
+                { IndentBraces, true },
+                { NewLinesForBracesInAnonymousMethods, true }
+            };
+
+            var code = @"
+delegate () 
+        {
+}";
+            var expected = @"
+delegate ()
+        {
+        }";
+            await AssertFormatAsync(expected, code, changedOptionSet: options);
+        }
```

Comment:
would like tests where curlies are on same line (a test where this is the same as the delegate-line, and a test where it is on the next line).  again wtih all four values.  thanks!